Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6
+FROM ubuntu:precise
```

Comment:
This `Dockerfile` is designed to mimic the TravisCI environment when running browser tests locally.